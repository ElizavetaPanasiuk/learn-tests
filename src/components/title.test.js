import { configure, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

import Title from './Title';

configure({
  adapter: new Adapter()
});

describe('<Title />', () => {
  it('should return component with piped props', () => {
    const component = shallow(<Title />);
    expect(toJson(component)).toMatchSnapshot();
  });

  it('should return component with default props', () => {
    const component = shallow(<Title title="test title" />);
    expect(toJson(component)).toMatchSnapshot();
  });
});