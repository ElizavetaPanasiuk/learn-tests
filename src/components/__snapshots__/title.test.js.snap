// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should return component with default props 1`] = `
<h1
  className="title"
>
  test title
</h1>
`;

exports[`<Title /> should return component with piped props 1`] = `
<h1
  className="title"
>
  Simple title
</h1>
`;
